plugins/janus_audiobridge.c

```diff
@@ -7767,9 +7781,24 @@ static void *janus_audiobridge_mixer_thread(void *data) {
 						}
 					}
 				}
+				if(p->annc->to || p->annc->exclude) {
+					janus_audiobridge_rtp_relay_packet *pkt = g_malloc(sizeof(janus_audiobridge_rtp_relay_packet));
+					pkt->data = g_malloc(sizeof(resampled));
+					memcpy(pkt->data, resampled, sizeof(resampled));
+					pkt->ssrc = 0;
+					pkt->timestamp = 0;
+					pkt->seq_number = 0;
+					pkt->silence = FALSE;
+					pkt->length = read;
+
+					janus_mutex_lock(&p->qmutex);
+					/* Insert packets sorting by sequence number */
+					p->inbuf = g_list_prepend(p->inbuf, pkt);
+					janus_mutex_unlock(&p->qmutex);
+				}
+
 				ps = ps->next;
 			}
-			g_list_free_full(anncs_list, (GDestroyNotify)janus_audiobridge_participant_unref);
```

Comment:
Why have you removed this?

Reply:
`anncs_list` is required later and moved below